avahi-daemon/static-services.c

```diff
@@ -308,12 +308,21 @@ typedef enum {
     XML_TAG_TXT_RECORD
 } xml_tag_name;
 
+typedef enum {
+    TXT_RECORD_VALUE_TEXT,
+    TXT_RECORD_VALUE_BINARY_HEX,
+    TXT_RECORD_VALUE_BINARY_BASE64
```

Comment:
Nitpick: Missing comma on the last element (reduces diff noise if further elements are added in future).